### DISCUSSION TITLE
Transfers to or from my Prime Saver account have posted order of `0`

### DISCUSSION THREAD

**Discussion by scholtz-gnome** [Investec-Developer-Community/.github#10] [Q&A]
Whether of `type` DEBIT or CREDIT, all transfers between my Prime Saver account and my general account have a `postedOrder` of `0`. Is this normal? What is the reason for this? They all have a `postingDate`, but they don't have a `valueDate`.

**Comment by programmable-banking-community** [comment]
Hey @scholtz-gnome, this is interesting. We'll check with the data team regarding this and get back to you as soon as we can. Thanks for the spot!

**Reply by scholtz-gnome** [reply]
Hey @programmable-banking-community, any feedback on this? I've noticed this to be an issue when transferring to or from my bond account as well, not just my Prime Saver. Having the `postedOrder` is very helpful for ordering transactions, since there could be multiple transactions per day and the date doesn't contain the actual time of the transaction.
